wordpress-mobile/WordPress-iOS
# Issue/3911 use video shortcode on self hosted sites

Fixes #3911 

To test:
 - Start the app
 - Select/Add a self-hosted site to the app (non-jetpack)
 - Create a new post using the Hybrid editor
 - Upload a video to the post
 - Switch to html mode, and check the [video] shortcode is being used
 - Save the post and publish
 - Check on the web (wp-admin) if the video shows up correctly on the post.

Needs review: @diegoreymendez
